pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -2234,6 +2234,26 @@ let
         };
       };
 
+      jackmacwindows.vscode-computercraft = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "vscode-computercraft";
+          publisher = "jackmacwindows";
+          version = "1.1.1";
+          hash = "sha256-ec1I3oQ06iMdSUcqf8yA3GjE7Aqa0PiLzRQLwFcL0KU=";
+        };
+        postInstall = ''
+          # Remove superflouous images to reduce closure size
+          rm $out/$installPrefix/images/*.gif
+        '';
+        meta = {
+          description = "Adds autocompletion for ComputerCraft/CC: Tweaked APIs to Visual Studio Code";
```

```suggestion
          description = "A Visual Studio Code extension for ComputerCraft auto-completion";
```
Would this fit?
Key here being "A Visual Studio Code extension ..."

Also, `changelog` is missing.

---

Fixed.